logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/LogoutUseCase.kt

```diff
@@ -74,17 +75,17 @@ internal class LogoutUseCaseImpl @Suppress("LongParameterList") constructor(
                 clearClientDataUseCase()
                 clientRepository.clearCurrentClientId()
                 clientRepository.clearHasRegisteredMLSClient()
+                // After logout we need to mark the Firebase token as invalid locally so that we can register a new one on the next login.
```

Comment:
@MohamadJaara Looking at the reasons should we also remove it from here? I assume that in that case we also clearing entire DB?

Reply:
no need to update it here since all of the local data will be wiped anyway